### PR TITLE
Added Actions For Updating Status

### PR DESCRIPTION
**Pull request by fr-rpathade**
#### Descriptions:
Added Test cases

#### What's New
  - Added following actions
  - Update Archived Issue
  - Update Issue Status
  - Update Archived Asset
  - Update ASN status to False Positive
  - Update Prefix status to False Positive
  - Update IP status to False Positive
  - Update Domain status to False Positive
  - Update Sub-Domain status to False Positive
  - Update Leaked Credential status

- Updated/fixed output schemas of get_domains, get_breaches_by_id.

easm-connector-test-result

### CODE REVIEW

**Comment by fr-rpathade** [comment]
@cs-prerna-joshi @cs-shahbaj-tamboli Please generate the playbooks.json file, I've not updated it.

File: fortinet-fortirecon-easm/assets.py [hunk 1]
```diff
@@ -101,15 +101,46 @@ def get_security_insights(config, params):
     return get_multiple_records(config=config, endpoint=endpoint)
 
 
-def get_security_insights(config, params):
```
Comment (cs-sameer-pathan) [inline fortinet-fortirecon-easm/assets.py:104]: Why this function is removed ?

File: fortinet-fortirecon-easm/info.json [hunk 2]
```diff
@@ -3713,6 +3720,261 @@
...
+            "ACTIVE",
+            "RESOLVED",
+            "FALSE_POSITIVE",
+            "RISK_ACCEPTED"
+          ]
+        }
+      ]
+    },
+    {
+      "operation": "update_asn_asset_status_to_false_positive",
```
Comment (cs-sameer-pathan) [inline fortinet-fortirecon-easm/info.json:3816]: Only one action can be implemented `Update Asset To False Positive` or `Mark Asset As False Positive`  with select parameter `Asset Type` with option `ASN`, `IP Prefix`, `Domain `, `Sub-Domain`.

File: fortinet-fortirecon-easm/info.json [hunk 3]
```diff
@@ -3713,6 +3720,261 @@
...
+          "description": "Specify the archived issue ID to update the status to active.",
+          "tooltip": "Specify the archived issue ID to update the status to active."
+        }
+      ]
+    },
+    {
+      "operation": "update_issue_status",
+      "title": "Update Issue Status",
+      "annotation": "update_issue_status",
+      "description": "Updates the status of the Security issue record.\n If the given issue status is set to false_positive or risk_accepted then issue will be visible in the archived issues.",
```
Comment (cs-sameer-pathan) [inline fortinet-fortirecon-easm/info.json]: Please use FALSE_POSITIVE and RISK_ACCEPTED values in description

File: fortinet-fortirecon-easm/info.json [hunk 4]
```diff
@@ -3713,6 +3720,261 @@
...
+          "visible": true,
+          "required": true,
+          "description": "Specify the archived asset ID to update the status to active.",
+          "tooltip": "Specify the archived asset ID to update the status to active."
+        }
+      ]
+    },
+    {
+      "operation": "update_archived_issue",
+      "title": "Update Archived Issue",
```
Comment (cs-sameer-pathan) [inline fortinet-fortirecon-easm/info.json]: Title can be updated to `Mark Archived Issue as Active`.

**Review by cs-sameer-pathan — CHANGES_REQUESTED** [review]

File: fortinet-fortirecon-easm/assets.py (hunk 1 above)
Comment (fr-rpathade) [reply fortinet-fortirecon-easm/assets.py:104]: This is moved to the assets.py, I have basically moved the multiple python files of like ip, asn, domain etc to assets.py

File: fortinet-fortirecon-easm/info.json (hunk 2 above)
Comment (fr-rpathade) [reply fortinet-fortirecon-easm/info.json:3816]: I've purposefully kept it separate, as it is write operation so I want customer to explicitly select the update action for each asset type.

File: fortinet-fortirecon-easm/info.json (hunk 3 above)
Comment (fr-rpathade) [reply fortinet-fortirecon-easm/info.json]: This is done

File: fortinet-fortirecon-easm/info.json (hunk 4 above)
Comment (fr-rpathade) [reply fortinet-fortirecon-easm/info.json]: This is done

**Review by cs-sameer-pathan — APPROVED** [review]

**Review by cs-sameer-pathan — APPROVED** [review]

**Pull request merged**